style.css

```diff
@@ -185,3 +182,389 @@ a:hover {
+  list-style-type: none;
+  color: #fff;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  box-sizing: border-box;
+  position: relative;
+  right: 30px;
+}
+
+.other-content-tags > li {
+  width: fit-content;
+  padding: 10px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.other-content-link {
+  background-color: #ff6b00;
+  color: white;
+  padding: 14px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+  top: 10px;
+}
+
+.about {
+  position: absolute;
+  width: 375px;
+  height: 1601px;
+  left: 0;
+  top: 4000px;
+  background: url("../Images/Frame.png") no-repeat;
+  background-color: #1c1a19;
+  background-position: 100%;
+  background-position-y: 23%;
+  align-items: center;
+}
+
+.about-primary-text {
+  width: 327px;
+  height: 79px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 20px;
+  margin-top: 120px;
+}
+
+.about-description {
+  width: 327px;
+  height: 148px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: 20px;
+  margin-top: -36px;
+}
+
+.about-link {
+  background-color: #ff6b00;
+  color: white;
+  padding: 14px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+  height: 40;
+  margin-left: 20px;
+}
+
+.about-hr {
+  width: 327px;
+  position: absolute;
+  border: 1px solid #fff;
+  margin-top: 3300px;
+  background: #ff6b00;
+}
+
+.about-bottom {
+  display: grid;
+  grid-template-columns: auto;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  bottom: 300px;
+  gap: 30px;
+  margin-top: 317px;
+  align-self: center;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.about-bottom-subtopic {
+  width: 150px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  text-align: center;
+  align-self: center;
+  justify-content: center;
+  color: #fff;
+  position: relative;
+  left: 35px;
+  top: 35px;
+}
+
+.about-bottom-items > ul > li {
+  width: 150px;
+  height: 40px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 100px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  list-style-type: none;
+  position: relative;
+  top: 24px;
+}
+
+.subtopic-language {
+  list-style-image: url("../Images/Rectangle55.png");
+}
+
+.subtopic-frameworks {
+  list-style-image: url("../Images/Rectangle56.png");
+}
+
+.subtopic-skills {
+  list-style-image: url("../Images/Rectangle57.png");
```

Comment:
[OPTIONAL] **There is an extra dot `.` in your image URL kindly make sure you keep one dot. 
Your code should look like this :arrow_down:**

` list-style-image: url("./Images/Rectangle55.png");`  **Notice that I removed one dot from the URL path**